src/enclave/rpccontext.h

```diff
@@ -16,7 +16,7 @@ namespace enclave
   {
     size_t client_session_id = InvalidSessionId;
     std::vector<uint8_t> caller_cert = {};
-    bool is_forwarded = false;
+    bool forward = false;
```

Comment:
I wrongly assumed that this flag indicated whether this context corresponded to a forwarded request (from the primary's point of view). It is actually about whether a request was this session was forwarded from the _backup_'s point of view. Please let me know if there's a better name for it.

Reply:
The flag reflects the fact that at least one command on the session has been forwarded, and is used to decide that further commands should be forwarded. It's orthogonal to the primary/backup point of view, it's a session-scope flag. If the node believes it is primary, it doesn't care about this flag.

Reply:
`is_forwarding` or `has_forwarded` may be clearer?

Reply:
Since it is a session scope flag it could be explicitly named to avoid confusion, i.e. `is_session_forwarded` or `is_session_forwarding`

Reply:
This is within a `SessionContext` object, so is generally accessed by something like `ctx->session.foo`, so it shouldn't need an additional `session` in its name.

Reply:
true :)

Reply:
It's a matter of perspective, if you think the frontend forwards [the requests for] the session, then is_forwarded makes sense. If you think the session is the subject, then is_forwarding. In practice, the code implements the former, but I don't feel strongly about this.

Alternatively, we could consider something like `process_locally` if we think that's clearer?

Reply:
Let's go for `is_forwarding`.